egs/aishell/s10/chain/feat_dataset.py

```diff
@@ -81,6 +97,8 @@ def __getitem__(self, i):
 
     def __str__(self):
         s = 'feats scp: {}\n'.format(self.feats_scp)
+        if self.ivector_scp:
```

Comment:
`self.ivector_scp` should be initialized to `None` at the begining of the `__init__` function.